File: 251004/Markovskii/NewsService/Domain/Entities/News.cs
﻿namespace Domain.Entities;

public class News
{
    public long Id { get; set; }
    public long EditorId { get; set; }
    public string Title { get; set; }
    public string Content { get; set; }
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }
}

File: 351004/Nepesov/Lab1/untitled/src/main/java/org/example/model/News.java
package org.example.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;
import java.util.ArrayList;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
@EqualsAndHashCode(callSuper = true)
public class News extends BaseEntity {
    private Long editorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
    // Для связи Many-to-Many со стикерами
    private List<Long> stickerIds = new ArrayList<>();
}